content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -330,14 +334,14 @@ YYYY/MM/DD 16:06:35 Datadog Tracer <version> DEBUG: Sending payload: size: <size
 {{% /tab %}}
 {{% tab ".NET" %}}
 
-### Profiler log
+### Native log
```

Comment:
Only the code is "native" or "managed," not the logs. Should we rename this? Technically they are "logs from native code" and "logs from managed code," but maybe that's too long. Open to suggestions.

Reply:
I changed it to "logs from native code" and "logs from managed code"